interface/forms/dictation/view.php

```diff
@@ -26,18 +28,26 @@
 <div class="container">
 <?php
 include_once("$srcdir/api.inc");
-$id = filter_input(INPUT_GET, 'id', FILTER_VALIDATE_INT);
+$id = filter_input(INPUT_GET, "id", FILTER_VALIDATE_INT);
 $obj = formFetch("form_dictation", $id);
 ?>
    <form method=post action="<?php echo $rootdir?>/forms/dictation/save.php?mode=update&id=<?php echo attr($id);?>" name="my_form">
      <div class="page-header">
       <h1><?php echo xlt('Speech Dictation Review/Edit'); ?></h1>
      </div>
      <div class="form-group">
-       <label for="dictation"><?php echo xlt('Dictation: '); ?></label><br><textarea class="form-control ckeditor" cols=80 rows=24 wrap="virtual" name="dictation" ><?php echo html_entity_decode(text($obj{"dictation"}));?></textarea>
+       <label for="dictation"><?php echo xlt('Dictation: '); ?></label><br><textarea class="form-control ckeditor" cols=80 rows=24 wrap="virtual" name="dictation" ><?php 
```

Comment:
Suggest breaking this super long line up. Textarea with "form-control" shouldn't have the colts attribute, CSS takes care of that